inbc-program/inbc/command/command.py

```diff
@@ -129,7 +129,7 @@ def invoke_update(self, args: Any) -> None:
 
         @param args: arguments passed to command-line tool.
         """
-        super()._send_manifest(args, RESTART_CHANNEL)
+        super()._send_manifest(args, RESTART_CMD_CHANNEL)
 
     def search_response(self, payload: Any) -> None:
         """Search for keywords in response message
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Maintainability$</h2>**
**Suggestion:** The search_response method in the RestartCommand class has a comment about the expected message format. This is a potential maintenance issue if the message format changes. Consider defining these expected messages as constants or using a configuration file to make updates easier and the code more maintainable.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
RESTART_EXPECTED_MESSAGE = "Device will restart after one minute."

def search_response(self, payload: Any) -> None:
    if search_keyword(payload, [RESTART_SUCCESS]):
        logger.info(RESTART_EXPECTED_MESSAGE)
        self.terminate_operation(COMMAND_SUCCESS, InbcCode.SUCCESS.value)
```